ciao-controller/command.go

```diff
@@ -247,6 +255,7 @@ func (c *controller) launchCNCI(tenantID string) error {
 		WorkloadID: workloadID,
 		TenantID:   tenantID,
 		Instances:  1,
+		Name:       "cnci",
```

Comment:
what if a user decides to name their instance "cnci"? how will we use this hostname?